_posts/2021-09-21-being-right-is-just-the-beginning.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Being Right Is Just the Beginning"
+date: 2021-09-21
+image: "being-right.jpeg"
+category: Security
+tags: infosec, conference
+author: leigh_hall
+summary: "As an industry we’re obsessed with being smart. And that’s ok - good even, in the right context. But to get what we want we need to stop being the smartest person in the room and start being the most helpful in the room"
+---
+
+On the 8th September I was very grateful to have the opportunity to speak at my local security/hacker conference DC151 in Leeds.
```

Comment:
Pity the website is gone - would be nice to link to something on DC151, even if just twitter account